cql3/type_json.cc

```diff
@@ -228,6 +230,18 @@ struct from_json_object_visitor {
     }
     bytes operator()(const boolean_type_impl& t) {
         if (!value.IsBool()) {
+            if (value.IsString()) {
+                std::string str(rjson::to_string_view(value));
+                boost::trim_all(str);
+                boost::to_lower(str);
+
+                if (str == "true") {
+                    return t.decompose(true);
+                }
+                else if (str == "false") {
```

Comment:
nitpick: our usual coding style is to put the else on the same line as the "}", to make it more obvious there is an else. This is documented (sort of... by example...) in Seastar's coding-style.md.